src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -121,7 +123,11 @@ class StencilTest extends GPUTest {
     this.runStencilStateTest(testStates, expectedColor);
   }
 
-  runStencilStateTest(testStates: TestStates[], expectedColor: Float32Array) {
+  runStencilStateTest(
+    testStates: TestStates[],
+    expectedColor: Float32Array,
+    isSingleEncoderMultiplePass?: boolean
```

Comment:
Can use defaulting here to make the type `boolean` instead of `boolean | undefined`.

```suggestion
    isSingleEncoderMultiplePass: boolean = false
```

Reply:
Fixed.